src/woo_publications/publications/tests/test_admin_logging.py

```diff
@@ -300,7 +301,9 @@ def test_admin_inline_create_admin(self):
 
         with self.subTest("update inline update logging"):
             document = Document.objects.get()
-            log = TimelineLogProxy.objects.filter(object_id=document.pk).get()
+            log = TimelineLogProxy.objects.filter(
+                object_id=str(document.pk), extra_data___cached_object_repr="title"
+            ).get()
```

Comment:
looks like it would be useful to have a `TimeLineLogProxy.objects.for_object(document)` method on the manager - feel free to make a PR to https://github.com/maykinmedia/django-timeline-logger/blob/master/timeline_logger/models.py